src/middleware/jira-symmetric-jwt-middleware.ts

```diff
@@ -33,12 +33,14 @@ export const jiraSymmetricJwtMiddleware = async (req: Request, res: Response, ne
 			req.log.warn({ err }, "Could not verify symmetric JWT");
 			return res.status(401).send("Unauthorised");
 		}
+
 		res.locals.installation = installation;
 		res.locals.jiraHost = installation.jiraHost;
-		req.session.jiraHost = installation.jiraHost;
-
-		// Check whether logged in user has Jira Admin permissions and save it to the session
-		await fetchAndSaveUserJiraAdminStatus(req, verifiedClaims, installation);
+		if (req.session) {
```

Comment:
create branch isnt inside a session, so if the session doesnt exist we dont want to try assign things to undefined!

Reply:
Hmm, have you try this? Shouldn't be the `req.session` is always can be assigned some value, even if it is undefined? I thought there's an interceptor/wrapper on that object by the middleware. Maybe I'll try it myself.

Reply:
I tried, seems you can always set things to session as long as the route is behind `RootRouter.use(cookieSessionMiddleware);`

---
Because that, I still prefer just set the jiraHost into session since it is verified.

Reply:
oooo nice